src/index.ts

```diff
@@ -191,21 +189,23 @@ export function graphqlHTTP(options: Options): Middleware {
   }
 
   return async function graphqlMiddleware(
-    request: $Request,
-    response: $Response,
+    request: Request,
+    response: Response,
   ): Promise<void> {
     // Higher scoped variables are referred to at various stages in the asynchronous state machine below.
-    let params: GraphQLParams;
+    let params: GraphQLParams | undefined;
     let showGraphiQL = false;
     let graphiqlOptions;
-    let formatErrorFn = formatError;
+    let formatErrorFn: (
+      error: GraphQLError,
+    ) => GraphQLFormattedError = formatError;
```

Comment:
Can it be `typeof formatError`?

Reply:
Thanks. Actually, `let formatErrorFn = formatError` is sufficient now. Had to specify the type before only because of differences in the return types.